### PR TITLE
changed some stuff

### PR DESCRIPTION
**Pull request by echarlotte612**
Media Query for main and user and leaderboard thing
background grid thing
footer sticky bottom thing
login redirect thing

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -21,34 +21,33 @@ function App() {
...
+        </header>
+        <div className="app-body">
+          <Switch>
+            <Route exact path="/" component={Landing} />
+            <Route exact path="/main" component={Mainpage} />
+            <Route exact path="/signup" component={Signup} />
+            <Route exact path="/about" component={About} />
+            <Route exact path="/quiz" component={Quiz} />
+            <Route exact path="/leaderboard" component={Leaderboard} />
+            <Route exact path="/UserHistory" component={UserHistory} />
```
Comment (tamjankowska) [inline frontend/src/App.js:44]: Should this be lower case?

Comment (echarlotte612) [reply frontend/src/App.js:44]: Don't know if it matters ?

Comment (echarlotte612) [reply frontend/src/App.js:44]: worked for me while testing anyway...

Comment (tamjankowska) [reply frontend/src/App.js:44]: Soz, just saw your replies. If ya say so ;)

**Review by tamjankowska — APPROVED** [review]

**Pull request merged**